types/migrations/20220714123910_edited_collections_2.sql

```diff
@@ -0,0 +1,6 @@
+-- Add migration script here
+create type collection_section as enum ('Intro', 'Body');
+
+alter table collection_chapter
+    add section collection_section;
+    
```

Comment:
Make sure there's a newline at the end of the file, and would you mind running `sqlfluff fix types/migrations/20220714123910_edited_collections_2.sql` to format it?